price-estimator/README.md

```diff
@@ -1,6 +1,8 @@
 ## API
 
-The api is documented with [OpenAPI](https://www.openapis.org/) in `openapi.yml`. It can be accessed in a rendered and interactive fashion on https://editor.swagger.io/ through `File -> Import file`.
+The api is documented with [OpenAPI](https://www.openapis.org/) in `openapi.yml` and hosted at https://price-estimate-docs.dev.gnosisdev.com/ .
```

Comment:
ubernit: I would make this an explicit auto-link and not rely on the link parsing:

```suggestion
The api is documented with [OpenAPI](https://www.openapis.org/) in `openapi.yml` and hosted at <https://price-estimate-docs.dev.gnosisdev.com/>.
```